PSix/FacebookService.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  FacebookService.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/16/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+
+class FacebookService {
+    
+    private static let FB_EVENT_ATTRIBUTES = "cover,name,id,start_time,description"
+    
+    static func getLoggedInUserId(callback: (fbId: String) -> ()) {
+        if FBSDKAccessToken.currentAccessToken() != nil {
+            FBSDKGraphRequest(graphPath: "me", parameters: ["fields": "id"]).startWithCompletionHandler() {
+                (connection, result, error) -> Void in
+                if let fbId = result["id"] as? String {
+                    callback(fbId: fbId)
+                }
+            }
+        }
+    }
+    
+    static func getFutureEventsCreatedByUser(user: User, failure: ((NSError?) -> ())?, success: (result: NSDictionary) -> ()) {
+        let params = [
+            "fields": FB_EVENT_ATTRIBUTES,
+            "since":  Int((NSDate.timeIntervalSinceReferenceDate() + NSTimeIntervalSince1970))
+        ]
+        if let fbId = user["facebookId"] as? String {
+            if FBSDKAccessToken.currentAccessToken() != nil {
```

Comment:
I can't. The first condition is an if let which is a declaration and the second is a condition. There's [a question about it in SO](http://stackoverflow.com/questions/25202770/using-the-swift-if-let-with-logical-and-operator).